### ISSUE TITLE
autonet.get_pytorch_model() function return a network different from autonet itself

### ISSUE THREAD

**Issue by zym604** [automl/Auto-PyTorch#21]
Hi, I find that the autonet prediction results are different from the result predicted by the pytorch model obtained from autonet using get_pytorch_model function. To be specific:
```res_autonet = autonet.predict(X_train)```
is different from:
```
model = autonet.get_pytorch_model()
model.eval()
X_train = X_train.astype(np.float32)
res_model = model(Variable(torch.from_numpy(X_train).cuda())).data.cpu().numpy()
```
![image](https://user-images.githubusercontent.com/9557009/68909437-394c4580-071c-11ea-955f-fcf4a257bcd9.png)
As can be seen in the third picture, the two predictions have different values, while they're supposed to be equal. 

The full code could be seen  as attached:

```
__author__ = "Max Dippel, Michael Burkart and Matthias Urban"
__version__ = "0.0.1"
__license__ = "BSD"

import os, sys
sys.path.append(os.path.abspath(os.path.join(__file__, "..", "..", "..")))
from autoPyTorch import AutoNetRegression
from autoPyTorch.data_management.data_manager import DataManager

# Note: You can write your own datamanager! Call fit train, valid data (numpy matrices) 
dm = DataManager()
dm.generate_regression(num_features=21, num_samples=1500)
X_train=dm.X
Y_train=dm.Y
X_valid=dm.X_train
Y_valid=dm.Y_train

# Note: every parameter has a default value, you do not have to specify anything. The given parameter allow a fast test.
autonet = AutoNetRegression(budget_type='epochs', min_budget=1, max_budget=9, num_iterations=1, log_level='info')

res = autonet.fit(X_train=X_train, Y_train=Y_train, X_valid=X_valid, Y_valid=Y_valid)

print(res)

res_autonet = autonet.predict(X_train)

##retrain
import numpy as np
import torch
from torch.autograd import Variable
model = autonet.get_pytorch_model()
model.eval()
#autonet.print_help()
X_train = X_train.astype(np.float32)
res_model = model(Variable(torch.from_numpy(X_train).cuda())).data.cpu().numpy()

#plot
import matplotlib.pyplot as plt
plt.figure(figsize=(10,7))
plt.subplot(221)
plt.plot(Y_train,res_model,'.')
plt.xlabel('True result')
plt.ylabel('Model result')
plt.subplot(222)
plt.plot(Y_train,res_autonet,'.')
plt.xlabel('True result')
plt.ylabel('Model result')
plt.subplot(223)
plt.plot(res_autonet,res_model,'.')
plt.xlabel('AutoNet result')
plt.ylabel('Model result')
```

**Comment by zym604** [comment]
I ask this question because the degree of difference is much larger in my case using my own data:
![image](https://user-images.githubusercontent.com/9557009/68909673-29813100-071d-11ea-9c4f-6c6ef9d70aef.png)
So I return to check the example code and find out that such a difference exists in it too. It's just the degree of difference not so much as my case.

**Comment by LMZimmer** [comment]
Note that the auto net pipeline also includes all data preprocessing such as normalization. Did you consider that?

**Comment by zym604** [comment]
> Note that the autonet pipeline also includes all data preprocessing such as normalization. Did you consider that?

I didn't, that's possible the reason.
